app/views/finders/show_all_content_finder.html.erb

```diff
@@ -26,20 +26,17 @@
     <div class="govuk-width-container">
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-          <%= render "govuk_publishing_components/components/heading", {
-            text: sanitize("Search <span class='govuk-visually-hidden'>all content on</span> GOV.UK"),
-            heading_level: 1,
-            font_size: "xl",
-            margin_bottom: 4,
-          } %>
-
           <div id="keywords" role="search" aria-label="Sitewide" data-ga4-change-category="update-keyword text">
             <%= render "govuk_publishing_components/components/search", {
               id: "finder-keyword-search",
               name: "keywords",
               type: 'search',
               value: result_set_presenter.user_supplied_keywords,
               disable_corrections: true,
+              label_size: "xl",
+              label_text: sanitize("Search <span class='govuk-visually-hidden'>all content on</span> GOV.UK"),
```

Comment:
Are we just using sanitize to appease an error here? Since it has no external input `.html_safe` would feel more idiomatic - though since that is often a bit smelly `safe_join` could be best:

```
safe_join(["Search ", tag.span("all content on", class: "govuk-visually-hidden"), " GOV.UK"])
```